src/Symfony/Component/Form/ChoiceList/Factory/CachingFactoryDecorator.php

```diff
@@ -84,10 +84,10 @@ public function getDecoratedFactory()
     /**
      * {@inheritdoc}
      *
-     * @param callable|Cache\ChoiceValue|null  $value  The callable or static option for
-     *                                                 generating the choice values
-     * @param callable|Cache\ChoiceFilter|null $filter The callable or static option for
-     *                                                 filtering the choices
+     * @param callable|string|Cache\ChoiceValue|null  $value  The callable or static option for
```

Comment:
This looks wrong to me. Where do we use arbitrary strings?

Reply:
strings are for property paths IIRC

Reply:
If that's the case, we should update the description accordingly IMO.

Reply:
That's because of PropertyAccessDecorator, which also accept `PropertyPath` btw.
This makes me think that:
1. `PropertyAccessDecorator` should maybe accept `PropertyPath` instead?
2. `CachingFactoryDecorator` should accept `mixed` here, because it shouldn't know anything about what it decorates.

WDYT?